src/pages/login/components/card.module.scss

```diff
@@ -0,0 +1,132 @@
+.card {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+	height: 80vh;
+  width: 50vw;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, rgba(0, 0, 0, 0.3) 0px 30px 60px -30px;
+  border-radius: 20px;
+
+  @media (max-width: 800px) {
+    height: 100vh;
+    width: 100vw;
+    border-radius: 0px;
+  }
+  
+  @media (max-height: 600px) {
+    height: 100vh;
+    width: 100vw;
+    border-radius: 0px;
+  }
+}
+
+.card form {
```

Comment:
nest this class in `.card `with `& form {...}`

Reply:
nesting classes is a good approach, I always use it but remember that there is a limit recommendation.

'As a general guide, keep nesting only three levels deep.' (https://blog.logrocket.com/native-css-nesting/#:~:text=Common%20guides%20in%20using%20nesting,-Avoid%20over%2Dnesting&text=As%20a%20general%20guide%2C%20keep,names%20as%20much%20as%20possible.).

You could find recommendation like this in many articles